app/jinja_filters.py

```diff
@@ -190,19 +190,38 @@ def setAttributes(dictionary, attributes):
 
 
 @blueprint.app_template_filter()
-def question_as_legend(question):
+def answers_require_legend(question):
     question_type = question['type']
 
     if question_type == 'MutuallyExclusive':
-        return True
+        return False
 
     answers = question['answers']
     more_than_one_question = len(answers) > 1
 
-    if more_than_one_question:
+    if more_than_one_question and question_type != 'DateRange':
         return True
 
-    answer = answers[0]
+    return False
+
+
+@blueprint.app_context_processor
+def answers_require_legend_processor():
+    return dict(answers_require_legend=answers_require_legend)
+
+
+@blueprint.app_template_filter()
+def answer_requires_legend(question):
+    if answers_require_legend(question):
+        return False
+
+    question_type = question['type']
+
+    if question_type == 'MutuallyExclusive':
+        return True
+
+    answer = question['answers'][0]
+
     if 'type' in answer and any(
         answer['type'] in answer_type
         for answer_type in ['Checkbox', 'Radio', 'Date', 'Duration']
```

Comment:
Not part of your PR but this is ugly Python. Would be much clearer as:

```Python
try:
    return answer['type'] in ['Checkbox', 'Radio', 'Date', 'Duration']
except KeyError:
    return False
```

Reply:
Please change it, for me :smile:

Reply:
```python     
return 'type' in answer and answer['type'] in ['Checkbox', 'Radio', 'Date', 'Duration']
```